src/Sulu/Bundle/MediaBundle/Markup/MediaTag.php

```diff
@@ -144,9 +152,46 @@ function ($attributes) {
         $result = [];
         foreach ($medias as $media) {
             $media['url'] = $this->mediaManager->getUrl($media['id'], $media['name'], $media['version']);
+            $dispositionType = $this->findMediaDispositionType($media);
+
+            if ($dispositionType) {
+                $media['url'] .= '&' . $dispositionType . '=true';
+            }
+
             $result[$media['id']] = $media;
         }
 
         return $result;
     }
+
+    /**
+     * Find disposition type by media MIME type.
+     *
+     * @param array $media
+     *
+     * @return string|null
+     */
+    private function findMediaDispositionType(array $media)
+    {
+        if (
+            !isset($media['mimeType']) ||
```

Comment:
Put the first line of the condition in the same line as the `if`, and please move the `||` to the beginning of the line, since it is quite an important information.